fuzzers/machxo3/120-clkdiv/1300/fuzzer.py

```diff
@@ -5,13 +5,13 @@
 
```

Comment:
is this machxo3 change intentional?

Reply:
Yes. Parameter was just not used. So it was cleanup